bukkit/build.gradle

```diff
@@ -7,6 +7,10 @@ dependencies {
     implementation 'space.arim.morepaperlib:morepaperlib:0.4.3'
     implementation 'me.lucko:commodore:2.2'
     implementation 'net.william278:AdvancementAPI:master-SNAPSHOT'
+    implementation "xyz.xenondevs.invui:invui-core:1.15"
+    implementation "xyz.xenondevs.invui:inventory-access-r14:1.15"
+    implementation 'io.github.skytasul:glowingentities:1.3'
+    implementation 'com.github.Emibergo02:RedisChat:main-SNAPSHOT'
```

Comment:
not sure why we're including your plugin as an include dep - if there's functionality being added related to this it should be spun off into a separate PR imo